converters/fromZigbee.js

```diff
@@ -381,6 +381,13 @@ const converters = {
             return {action: 'off'};
         },
     },
+    identify: {
+        cluster: 'genIdentify',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options) => {
+            return {identify: 'identify'};
```

Comment:
Could you change this to action: identify (otherwise it will be cached and present on all future messages).

Reply:
done